### PR TITLE
Aegis 218

### PR DESCRIPTION
**Pull request by loahou04**
@TanujaG @richweclg @marcdonovan

### CODE REVIEW

**Comment by richweclg** [comment]
approved

File: main.js [hunk 1]
```diff
@@ -59,6 +53,31 @@ class NotificationComponent {
 			console.log(error);
```
Comment (marcdonovan) [inline main.js]: should remove console.*

**Comment by marcdonovan** [comment]
approve

File: main.js (hunk 1 above)
Comment (loahou04) [inline main.js]: This needs to be changed completely to actually do something.  Right now we aren't really doing anything with errors...we may want to throw down a "error" view of the notification header or something.  you are right it shouldnt be logged error but for now that is all we have...

**Pull request merged**